backend/package.json

```diff
@@ -30,9 +30,11 @@
   "license": "ISC",
   "dependencies": {
     "and": "^0.0.3",
+    "bcrypt": "^5.1.1",
     "body-parser": "^1.20.2",
     "chai-http": "^4.4.0",
     "cors": "^2.8.5",
+    "dcrypt": "^0.0.2",
```

Comment:
Why is `dcrypt` added in package.json? are we really using it?

Reply:
I'm sorry about that one, I might have made typo while installing 'bcrypt' package.